src/lib/externalAttachment/index.ts

```diff
@@ -0,0 +1,85 @@
+import { singleton } from 'tsyringe'
+import { Logger } from 'pino'
+import { logger } from '../logger.js'
+import Identity from '../identity.js'
+import env from '../../env.js'
+import { AttachmentRow } from '../db/types.js'
+
+@singleton()
+export class ExternalAttachmentService {
+  private log: Logger
+
+  constructor(private identity: Identity) {
+    this.log = logger.child({ service: 'ExternalAttachment' })
+  }
+
+  async getOidcConfig(oidcConfigUrl: string) {
+    const response = await fetch(`${oidcConfigUrl}/sequence/.well-known/openid-configuration`)
```

Comment:
this is definitely wrong. You're asuming the realm is `sequence`. Instead the  `oidcConfigUrl` should store the full path